packages/database/src/utilities/markRecordsForResync.js

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
Great util!

Reply:
You wrote it actually - I just extracted it! 🙌

Reply:
Nice extraction - it's a good idea as it will probably be used again in future :-)